GDJS/Runtime/events-tools/commontools.js

```diff
@@ -106,16 +106,38 @@ gdjs.evtTools.common.logicalNegation = function(bool) {
   return !bool;
 };
 
+/**
+ * Limit an value in a range
+ * @param {number} x Value.
+ * @param {number} min Minimum value.
+ * @param {number} max Maximum value.
+ * @returns {number} The new value.
```

Comment:
This is also a little inconsistency: you included a "the" where you didn't for the param tags